### PR TITLE
Sankey: include label option in optionChangesMap  (T1243009)

### PR DESCRIPTION
**Pull request by nikkithelegendarypokemonster**

### CODE REVIEW

File: packages/devextreme/testing/tests/DevExpress.viz.sankey/sankey.label.tests.js [hunk 1]
```diff
@@ -224,6 +224,26 @@ QUnit.test('Labels offsets', function(assert) {
     assert.deepEqual(yDifference, [30, 30, 30], 'vertical offset applied');
 });
 
+QUnit.test('label horizontalOffset option should be updated and re-painted on runtime. (T1243009)', function(assert) {
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.viz.sankey/sankey.label.tests.js]:
```suggestion
QUnit.test('label horizontalOffset option runtime change should update the layout (T1243009)', function(assert) {
```

File: packages/devextreme/testing/tests/DevExpress.viz.sankey/sankey.label.tests.js [hunk 2]
```diff
@@ -224,6 +224,26 @@ QUnit.test('Labels offsets', function(assert) {
...
+    const instance = createSankey({
+        dataSource: [{ source: 'A', target: 'Z', weight: 1 }, { source: 'B', target: 'Z', weight: 1 }],
+        label: {
+            horizontalOffset: 0,
+        }
+    });
+
+    const x1 = this.labels().map(function(label) { return label.children[0].attr.lastCall.args[0].translateX; });
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.viz.sankey/sankey.label.tests.js]:
```suggestion
    const initialX = this.labels().map(function(label) { return label.children[0].attr.lastCall.args[0].translateX; });
```

File: packages/devextreme/testing/tests/DevExpress.viz.sankey/sankey.label.tests.js [hunk 3]
```diff
@@ -224,6 +224,26 @@ QUnit.test('Labels offsets', function(assert) {
...
+
+    instance.option('label', {
+        horizontalOffset: 30,
+    });
+
+    const x2 = this.labels().map(function(label) { return label.children[0].attr.lastCall.args[0].translateX; });
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.viz.sankey/sankey.label.tests.js]:
```suggestion
    const updatedX = this.labels().map(function(label) { return label.children[0].attr.lastCall.args[0].translateX; });
```

Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.viz.sankey/sankey.label.tests.js]: Don't you want to create a getter function to reduce copypaste?
235 and 241 row is almost the same at the moment

Comment (nikkithelegendarypokemonster) [reply packages/devextreme/testing/tests/DevExpress.viz.sankey/sankey.label.tests.js]: i will try

Comment (nikkithelegendarypokemonster) [reply packages/devextreme/testing/tests/DevExpress.viz.sankey/sankey.label.tests.js]: [5c2f6d9](https://github.com/DevExpress/DevExtreme/pull/27756/commits/5c2f6d924fc31fa76ad46902858feee53ba83ee5)

Comment (nikkithelegendarypokemonster) [reply packages/devextreme/testing/tests/DevExpress.viz.sankey/sankey.label.tests.js]: [5c2f6d9](https://github.com/DevExpress/DevExtreme/pull/27756/commits/5c2f6d924fc31fa76ad46902858feee53ba83ee5)

File: packages/devextreme/testing/tests/DevExpress.viz.sankey/sankey.label.tests.js (hunk 2 above)
Comment (nikkithelegendarypokemonster) [reply packages/devextreme/testing/tests/DevExpress.viz.sankey/sankey.label.tests.js]: [5c2f6d9](https://github.com/DevExpress/DevExtreme/pull/27756/commits/5c2f6d924fc31fa76ad46902858feee53ba83ee5)

File: packages/devextreme/testing/tests/DevExpress.viz.sankey/sankey.label.tests.js (hunk 1 above)
Comment (nikkithelegendarypokemonster) [reply packages/devextreme/testing/tests/DevExpress.viz.sankey/sankey.label.tests.js]: [5c2f6d9](https://github.com/DevExpress/DevExtreme/pull/27756/commits/5c2f6d924fc31fa76ad46902858feee53ba83ee5)

**Review by Zedwag — APPROVED** [review]

**Review by ksercs — APPROVED** [review]

**Pull request merged**